compose-builder/Makefile

```diff
@@ -822,9 +822,9 @@ ifeq (taf-secty, $(filter taf-secty,$(ARGS)))
 	# taf has its special place holder from taf-device-services-mods and thus we need to keep it
 	# and extend security related things on top of it
 	ds_virtual_ext:= $(shell GEN_EXT_DIR="$(GEN_EXT_DIR)" ./gen_secure_compose_ext.sh device-virtual \
-		device-virtual device-virtual ' -cp=consul.http:\/\/edgex-core-consul:8500 --registry --confdir=CONF_DIR_PLACE_HOLDER')
+		device-virtual device-virtual ' -cp=consul.http:\/\/edgex-core-consul:8500 --registry --configDir=CONF_DIR_PLACE_HOLDER')
 	ds_modbus_ext:= $(shell GEN_EXT_DIR="$(GEN_EXT_DIR)" ./gen_secure_compose_ext.sh device-modbus \
-		device-modbus device-modbus ' -cp=consul.http:\/\/edgex-core-consul:8500 --registry --confdir=CONF_DIR_PLACE_HOLDER')
+		device-modbus device-modbus ' -cp=consul.http:\/\/edgex-core-consul:8500 --registry --configDir=CONF_DIR_PLACE_HOLDER')
```

Comment:
Is there a reason we need a CONF_DIR_PLACE_HOLDER here? 
Use of `CONF_DIR_PLACE_HOLDER` should be renamed to `CONFIG_DIR_PLACE_HOLDER` here and I think other files.

Reply:
`CONF_DIR_PLACE_HOLDER` is replaced by a variable `${CONF_DIR}` in TAF. Will need to update [the TAF script](https://github.com/edgexfoundry/edgex-taf/blob/03ddac7a5263b1ef5c76871b10e0b56880d07258/TAF/utils/scripts/docker/get-compose-file.sh#L37) as well.

Reply:
I see.  We can keep those part for the TAF compose files.
@cherrycl for awareness

Reply:
The rename of `CONFIG_DIR_PLACE_HOLDER` will cause the TAF script change.
`${CONF_DIR}` in TAF should also be renamed to `${CONFIG_DIR}`